metrics/heartbeat.go

```diff
@@ -0,0 +1,13 @@
+package metrics
+
+// HeartBeatMetrics is a collection of metrics used in hearbeats
+type HeartBeatMetrics struct {
+	NotifierIsAlive Meter
```

Comment:
maybe we could make this field private and add method `MarkNotifierIsAlive(bool)`?